### DISCUSSION TITLE
English Text Opening

### DISCUSSION THREAD

**Discussion by Wandering-Heiho** [dackst/nayuta#6] [Show and tell]
So I just recently finished editing the opening movie. I tested inserting the file by replacing the original nyt_op.pmf file located in the movie folder with the one in this post and it worked fine for me.

This is what the section with the logo now looks like.

https://user-images.githubusercontent.com/73907314/103162178-2fb85c80-47bb-11eb-9c5c-7dfe449d3202.mp4

Here's the pmf file to replace the original located in the movie folder.
[Nayuta_no_Kiseki-English_Opening.zip](https://github.com/dackst/nayuta/files/5743972/Nayuta_no_Kiseki-English_Opening.zip)

_Happy Holidays!_

**Comment by dackst** [comment]
Thanks for taking the time to do this. It's nice that you were able to replicate the effect with the shooting star. 

I'll include it in a new release once I finish going through the game again to collect some text adjustments.

**Reply by Wandering-Heiho** [reply]
Glad to be of help, I had fun working on it.

I've also starting making preparations for the After Story credits to make the character names match up with whats used in the rest of the game. While elements to edit are less complex than what needed to be done for the opening, the amount of footage that needs to be edited is much more due to it being slow moving credits.

**Reply by dackst** [reply]
I'm not sure how much you have already done, and you can either or both or none if you want, but I think it would be more worth doing the "Epilogue" credits, as those have (non-scrolling) Japanese text, compared to the After Story's, which is already in English in some form.

**Reply by Wandering-Heiho** [reply]
Understood, I'll see what I can do with the Epilogue's credits.
I did a quick check using DeepL to figure out what the text was roughly about. Turns out it's the cast list like at the beginning of the After Story credits, just in Japanese.
I'll try to remove the Japanese text and replace it with English text. But if I run into difficulties with that (like if I run into problems with the parts that have the planet go behind the text), I might just leave the Japanese text there and have the English text appear next to it.
